mount/losetup_linux.go

```diff
@@ -84,6 +105,32 @@ func setupLoopDev(backingFile, loopDev string, param LoopParams) (_ *os.File, re
 		}
 	}()
 
+	if ok, err := kernelversion.GreaterEqualThan(
+		kernelversion.KernelVersion{Kernel: 5, Major: 8}); err == nil && ok {
+		config := LoopConfig{
+			Fd: uint32(back.Fd()),
+		}
+
+		copy(config.Info.File_name[:], backingFile)
```

Comment:
not related to this pr. I think we should check `backingFIle` length before copy.
We can handle it in the follow-up pull request.

Reply:
copy truncates the input if is longer than the destination. What would be the desired behavior?

Reply:
I think if the backingFile name is long than 64, we should return error. 
We don't need to add the enhancement in this pr.

Reply:
I don't think it is needed, that field is only for information purposes, we should not fail loop device setup because the path provided is longer than 64 characters.